internal/configuration/validator/access_control_test.go

```diff
@@ -35,6 +36,29 @@ func (suite *AccessControl) TestShouldValidateCompleteConfiguration() {
 	suite.Assert().False(suite.validator.HasErrors())
 }
 
+func (suite *AccessControl) TestShouldValidateEitherDomainsOrDomainsRegex() {
```

Comment:
We can't bypass with subject information, minimum is 1FA since anonymous users have no subject information. I believe I added a check for this, but I'll ensure that I did before I resolve this issue.

Reply:
Confirmed. Configurations which contain the subexp names used for this cause an error on startup if the policy is bypass:

```go
func validateBypass(rulePosition int, rule schema.ACLRule, validator *schema.StructValidator) {
	if len(rule.Subjects) != 0 {
		validator.Push(fmt.Errorf(errAccessControlRuleBypassPolicyInvalidWithSubjects, ruleDescriptor(rulePosition, rule)))
	}

	for _, pattern := range rule.DomainsRegex {
		if utils.IsStringSliceContainsAny(authorization.IdentitySubexpNames, pattern.SubexpNames()) {
			validator.Push(fmt.Errorf(errAccessControlRuleBypassPolicyInvalidWithSubjectsWithGroupDomainRegex, ruleDescriptor(rulePosition, rule)))
			return
		}
	}
}
```